test/integration/host_js_test.rb

```diff
@@ -167,7 +172,7 @@ class HostJSTest < IntegrationTestWithJavascript
         loop while find(:css, '#interfaceModal', :visible => false).visible?
       end
       click_button 'Submit' #create new host
-      wait_for_ajax
```

Comment:
Should we keep this in?  It might help wait for some of the graphs on the host#show page.